cache/cache.cpp

```diff
@@ -151,10 +151,9 @@ read_response_t cache_manager::read(const unsigned char *id, uint64_t ioflags) {
 	return m_caches[idx(id)]->read(id, ioflags);
 }
 
-int cache_manager::remove(const dnet_cmd *cmd,
-                          ioremap::elliptics::dnet_remove_request &request,
```

Comment:
`dnet_remove_request` is no longer needed?

Reply:
Client uses `dnet_remove_request`.